plugins/workunitservices/CMakeLists.txt

```diff
@@ -57,7 +60,8 @@ target_link_LIBRARies ( workunitservices
          hrpc 
          remote 
          dalibase 
-         workunit 
+         workunit
+         SMCLib
```

Comment:
I assume this is for getSashaServiceEP ? Not sure it's a sensible structure though. Means runtime stuff is dependent on ESP dlls. Is there not somewhere common that the function could go?

Reply:
I've added a helper func (getService) to a new module in thorhelper.
This will be used by this plugin and by engines.